messages/3.0.0.md

```diff
@@ -0,0 +1,206 @@
+# MarkdownTOC - 3.0.0
+
+---
+
+**NOTE**
+This upgrade contains some **breaking changes**. 
```

Comment:
Suggestion: "This upgrade contains **breaking changes**. The fact is more relevant, than the amount.

Reply:
I couldn't understand well, sorry...

Which is your suggestion?

1:
This upgrade contains some **breaking changes**.
↓
This upgrade contains **breaking changes**.

or

2:
This upgrade contains some **breaking changes**.
↓
This upgrade contains **breaking changes**. The fact is more relevant, than the amount.

Reply:
Loose the "some", I would go for: "This upgrade contains breaking changes."

Reply:
OK, I got it!